Tests/NetworkingTests/ErrorProcessorTests.swift

```diff
@@ -129,7 +129,7 @@ extension ErrorProcessorTests {
     
     func test_apiManager_request_originalErrorShouldRemain() async {
         let apiManager = APIManager(
-            urlSession: URLSession.shared,
+            responseProvider: URLSession.shared,
```

Comment:
As mentioned in `APIManager` I would rather see the `urlSession` parameter name here